package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclear",
-  "version": "0.6.9",
+  "version": "0.6.10",
```

Comment:
Can you remove this file from the commit or change to correct version?

Reply:
Ok I'm going to try to amend the previous commit then.

Reply:
ok thanks, I won't. I'm sorry for asking but could you (@nukeop) or @HaiDang666 take a look at https://github.com/nukeop/nuclear/pull/936#issuecomment-837023836 please ? Thank you in advance.

Reply:
Btw, please don't **delete** the package-lock, just don't include it in your pull request.

Reply:
I think that reverting the changes to this package-lock.json will also fix the CI errors.

Reply:
> 
> 
> I think that reverting the changes to this package-lock.json will also fix the CI errors.

@nukeop I can't seem to get the project to start, I already tried cloning it again but it always gives me the same error as in the CI. Given that I don't have time a lot of time available to me due to having to perform college work, including my Master Thesis, I would suggest closing this PR, and hopefully one day when I'm more free I'll try to open a new PR, with the suggested changes already included.

Sorry for all the trouble.